Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -97,6 +98,13 @@ class HMTLNodeToNSAttributedString: SafeConverter {
     /// - Returns: the converted node as an `NSAttributedString`.
     ///
     fileprivate func convertCommentNode(_ node: CommentNode, inheritingAttributes inheritedAttributes: [String:Any]) -> NSAttributedString {
+        if node.comment == "more" {
+            var attributes = inheritedAttributes;
+            let moreAttachment = MoreAttachment()
+            moreAttachment.message = NSAttributedString(string: NSLocalizedString("MORE", comment:"Text for the center of the   more divider"), attributes: defaultAttributes)
```

Comment:
There are extra spaces here in the `comment` parameter in the `NSLocalizedString` call.